MetovaTestKit/UIKitTesting/UIAlertControllerAssertions.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  UIAlertControllerAssertions.swift
+//  MetovaTestKit
+//
+//  Created by Logan Gauthier on 6/22/17.
+//  Copyright © 2017 Metova. All rights reserved.
+//
+//  MIT License
+//
+//  Permission is hereby granted, free of charge, to any person obtaining
+//  a copy of this software and associated documentation files (the
+//  "Software"), to deal in the Software without restriction, including
+//  without limitation the rights to use, copy, modify, merge, publish,
+//  distribute, sublicense, and/or sell copies of the Software, and to
+//  permit persons to whom the Software is furnished to do so, subject to
+//  the following conditions:
+//
+//  The above copyright notice and this permission notice shall be
+//  included in all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+//  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+//  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+//  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+//  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+//  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+//  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import Foundation
+
+// MARK: - Assertions
+
+/// Test that a view controller presented an alert with the specified title, message, action titles, and action styles.
+///
+/// - Parameters:
+///   - presenter: The view controller expected to have presented the alert.
+///   - style: The expected style of the alert.
+///   - title: The expected title of the alert.
+///   - message: The expected message of the alert.
+///   - actions: The list of expected actions in the expected order.
+///   - failureMessage: The message to log upon failure.
+///   - file: The name of the file to report the failure for. The default value will report the file from which this method is called.
+///   - line: The line number to report the failure for. The default value will report the line number of the calling site.
+public func MTKAssertAlertIsPresented(by presenter: UIViewController, style: UIAlertControllerStyle, title: String?, message: String?, actions: [ExpectedAlertAction], _ failureMessage: String? = nil, file: StaticString = #file, line: UInt = #line) {
+    
+    guard let alert = presenter.presentedViewController as? UIAlertController else {
```

Comment:
A view controller can only present one view controller at a time. If you try to present another view controller while already presenting, it will fail and a warning will be logged to the console.